_includes/code/wcs.authentication.api.key.mdx

```diff
@@ -3,7 +3,34 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <Tabs groupId="languages">
-<TabItem value="py" label="Python">
+<TabItem value="py4" label="Python (v4)">
+
+```python
+import weaviate
+
+# Connect to a WCS instance
+import os
+
+client = weaviate.connect_to_wcs(
+    cluster_url=os.getenv("WCS_DEMO_URL"),  # Set this environment variable
```

Comment:
This instruction sounds as through they need to leave the `WCS_DEMO_URL` env var name and then set it. 

How about:
```python
cluster_url="<YOUR_WCS_URL>",  # Set this, or use an env var, e.g. os.getenv("WCS_URL")
```

And similar for the line below